adapters/coap-vertx-base/src/test/java/org/eclipse/hono/adapter/coap/CoapOptionInjectExtractAdapterTest.java

```diff
@@ -39,17 +44,46 @@ class CoapOptionInjectExtractAdapterTest {
     @Test
     public void testJaegerTracerInjectsAndExtractsSpanContext() {
 
-        final Configuration config = new Configuration("test");
-        final Tracer tracer = config.getTracer();
-        final Span span = tracer.buildSpan("do").start();
+        // explicitly using Jaeger tracer here - the OpenTelemetry TracerShim doesn't support BINARY propagation yet (see https://github.com/open-telemetry/opentelemetry-specification/issues/437)
+        final Configuration jaegerConfig = new Configuration("test");
+        final Tracer jaegerTracer = jaegerConfig.getTracer();
+        final Span span = jaegerTracer.buildSpan("do").start();
```

Comment:
FMPOV we should remove support for binary from the CoAP adapter anyway. I had already started working on this ...

Reply:
In order to support binary, we need to define a CoAP option number that is used to contain the encoded tracing context. However, the name/number of that option would always be Jaeger specific and with the advent of W3C Trace Context and the standard header names defined there, it seems to be more reasonable to apply for standard CoAP option numbers for these headers.

Reply:
Ok, I've now removed the binary propagation support here.